crypto/asn1/asn1_lib.c

```diff
@@ -411,8 +411,9 @@ unsigned char *ASN1_STRING_data(ASN1_STRING *x)
 }
 #endif
 
-char *sk_ASN1_UTF8STRING2text(STACK_OF(ASN1_UTF8STRING) *text, const char *sep,
-                              size_t max_len /* excluding NUL terminator */)
+char *ossl_sk_ASN1_UTF8STRING2text(STACK_OF(ASN1_UTF8STRING) *text,
```

Comment:
I would drop `sk_`...

Reply:
> I would drop `sk_`...

That seems wrong to me as that really works with stack and not a single ASN1_UTF8STRING.

Reply:
I will leave it as it is..